griptape/events/base_chunk_event.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from attrs import define, field
+
+from griptape.events.base_event import BaseEvent
+
+if TYPE_CHECKING:
+    from griptape.common import BaseDeltaMessageContent
+
+
+@define
+class BaseChunkEvent(BaseEvent):
+    token: str = field(metadata={"serializable": True})
```

Comment:
I don't think this should live at the top. I'd prefer we move it to `TextChunkEvent` and then introduce `partial_input` to `ActionChunkEvent`.

Reply:
do we not think its valuable to be able to drop this class in where `CompletionChunkEvent` lived before?

Reply:
If users are subscribing to the events directly they'll already need to make changes so I'm not too worried about backwards compatibility with fields. Also, the `Stream` will take care of this for them.